streamlit_drawable_canvas/frontend/src/DrawableCanvas.tsx

```diff
@@ -148,7 +147,7 @@ const DrawableCanvas = ({ args }: ComponentProps) => {
 
     canvas.on("mouse:up", (e: any) => {
       saveState(canvas.toJSON())
-      if (e["button"] === 3 && drawingMode === "polypath") {
+      if (e["button"] === 3) {
```

Comment:
Enable right-click to update the canvas info.

I got the values from `console.log()`, but I am not sure why the right-click value is "3" here but "2" in `lib/*.ts` files.